### PR TITLE
Add optional `EXPLAIN` to `TCK` tests

### PR DESCRIPTION
**Pull request by darrellwarde**
# Description

Adds an optional `EXPLAIN` to TCK tests via the `VERIFY_TCK` environment variable (set to anything) - verifies the validity of the generated Cypher.

### CODE REVIEW

**Review by angrykoala — APPROVED** [review]

File: .github/workflows/reusable-api-library-tests.yml [hunk 1]
```diff
@@ -45,10 +45,10 @@ jobs:
       - name: Overwrite GraphQL version
         run: yarn up --exact graphql@${{ matrix.graphql-version }}
       - name: Run TCK tests
-        run: yarn --cwd packages/graphql run test:tck --coverage
+        run: yarn --cwd packages/graphql run test tck --coverage
```
Comment (mjfwebb) [inline .github/workflows/reusable-api-library-tests.yml:48]: Is this an intentional change?

**Comment by angrykoala** [comment]
I think I found the leak issue. Basically we were creating a neo4j driver without closing it when the TCK test where ran with `VERIFY_TCK`. I changed it so the driver is created and destroyed on each `translate` operation. We can probably improve this with global hooks to avoid re-creating the driver so many times, but at least this seems to fix the memory leak

**Comment by darrellwarde** [comment]
> I think I found the leak issue. Basically we were creating a neo4j driver without closing it when the TCK test where ran with `VERIFY_TCK`. I changed it so the driver is created and destroyed on each `translate` operation. We can probably improve this with global hooks to avoid re-creating the driver so many times, but at least this seems to fix the memory leak

Hm, I thought because I had defined the `neo4j` variable outside of the function, that this wouldn't be the case! Turns out I may have been wrong. 🙃

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-2068.surge.sh

**Pull request merged**